src/Kmd.Logic.Email.Client/EmailOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Kmd.Logic.Email.Client
+{
+    /// <summary>
+    /// Provide the configuration options for using the Email service.
+    /// </summary>
+    public sealed class EmailOptions
+    {
+        /// <summary>
+        /// Gets or sets the Logic Email service.
+        /// </summary>
+        /// <remarks>
+        /// This option should not be overridden except for testing purposes.
+        /// </remarks>
+        public Uri EmailServiceUri { get; set; } = new Uri("https://gateway.kmdlogic.io/email/v1");
```

Comment:
For the time being should we keep the default gateway as shared dev, till we keep on adding methods to client so that by mistake we won't be testing on production? @avin101  what do you think?